docs/ml-classification-regression.md

```diff
@@ -125,7 +117,6 @@ Continuing the earlier example:
 [`LogisticRegressionTrainingSummary`](api/python/pyspark.ml.html#pyspark.ml.classification.LogisticRegressionSummary)
 provides a summary for a
 [`LogisticRegressionModel`](api/python/pyspark.ml.html#pyspark.ml.classification.LogisticRegressionModel).
```

Comment:
Shall we just add a short line to the `Example` section of MLoR:

"The following example shows how to train a multiclass logistic regression model with elastic net regularization, as well as extract the multiclass training summary." 

or something like that.

Reply:
Done.